doc/whatsnew/2.0.rst

```diff
@@ -49,3 +49,13 @@ Removed Changes
   issue, since the pathological case would have happen when the
   code was parsed by Python as well, without actually reaching the
   runtime step and as such, we decided to remove the error altogether.
+
+
+Added features
+==============
+
+* Refactoring message, 'consider-merging-isinstance', which is
+  used to check the type of the same object, with different calls, in the
+  same chained boolean operation.
+  Consider merging instance `isinstance(var1, (int, long))` instead of
```

Comment:
This needs an actual example and please remove the last part. So something as:

    .. code-block:: python

       $ cat a.py
       isinstance(x, int) or isinstance(z, int)
       $ pylint a.py -rn
       # pylint's output here

Also, modify the description to say a similar thing as the one from the ChangeLog. Right now I think it is a bit confusing.